### PR TITLE
Render amenity=internet_cafe

### PR DESCRIPTION
**Pull request by jeisenbe**
Fixes [#3451](https://github.com/gravitystorm/openstreetmap-carto/issues/3451)

Changes proposed in this pull request:

Adds rendering for amenity=internet_cafe on nodes and closed ways ("areas") with an icon based on the '@' symbol in "amenity brown" color. The icon and name label are rendered at z17 and up, similar to other amenities. Thanks @Tomasz-W for the icon.

Test rendering with links to the examples:

z18 Node with amenity=internet_cafe
https://www.openstreetmap.org/node/5130908200

z19 Node with amenity=internet_cafe

z18 Closed way (area) with amenity=internet_cafe
https://www.openstreetmap.org/way/436919789
internet-germany

### CODE REVIEW

File: amenity-points.mss [hunk 1]
```diff
@@ -3084,4 +3092,4 @@
       trunk/marker-height: 6;
     }
   }
-}
+}
```
Comment (kocio-pl) [inline amenity-points.mss]: It looks like you have accidentally removed new line at the end of file, this is technical detail, but please bring it back.

**Comment by kocio-pl** [comment]
Thanks, this PR looks good in general and I will test it soon, just please resolve a tiny code detail.

**Comment by jeisenbe** [comment]
I added back the final line on amenity-points.mss
Sorry about the large number of commits, next time I'll make sure to do the testing first.

**Comment by kocio-pl** [comment]
Thanks, my check has showed that it works good. Congratulations on you first PR! :smile:

**Pull request merged**